tools/api-markdown-documenter/src/utilities/ApiItemUtilities.ts

```diff
@@ -39,8 +39,32 @@ import type { Logger } from "../Logging.js";
 
 /**
  * This module contains general `ApiItem`-related types and utilities.
+ * @remarks Note: the utilities here should not require any specific context or configuration.
  */
 
+/**
+ * Represents "valid" API item kinds. I.e., not `None`.
+ *
+ * @public
+ */
+export type ValidApiItemKind = Exclude<ApiItemKind, ApiItemKind.None>;
+
+/**
+ * Gets the {@link ValidApiItemKind} for the provided API item.
+ *
+ * @throws If the item's kind is "None".
+ */
+export function getApiItemKind(apiItem: ApiItem): ValidApiItemKind {
+	switch (apiItem.kind) {
+		case ApiItemKind.None: {
```

Comment:
Curious: what would have an item kind of `None`?

Reply:
For our purposes, nothing, and would signal that we encountered corrupted input or something. My guess as to why it exists is for use in the various mixin "ApiItem" types they export - they aren't _real_ API item types, so they would be tagged as `None`.

Reply:
No, I take that back. They don't use a "kind" at all. Searching Rushstack's repo, I see no hits for `ApiItemKind.None` at all... So I actually have no idea why that type exists. Something for me to ask Pete about at some point.